hoomd/pytest/test_box_resize.py

```diff
@@ -0,0 +1,128 @@
+from math import isclose
+import numpy as np
+import numpy.testing as npt
+import pytest
+import hoomd
+
+
+@pytest.fixture(scope="module")
+def fractional_coordinates(n=10):
+    """
+    TODO: Does `numpy_random_seed()` in conftest.py run for this function?
```

Comment:
I don't know. That fixture is set with `@pytest.fixture(scope='function', autouse=True)` so it will be called before every function, but I don't know how that works with a module scope fixture.

You could make sure that it works in the correct order by making this a function scope fixture and include numpy_random_seed as an argument to ensure that that fixture is called first.

Reply:
I want to note that the linear_volume box resize test is still failing for one of the systems. In addition, the other variant tests pass when the xy for sys21 is set to 0.9 (It fails when it is 2). I still do not know why this is happening.

Reply:
@joaander for any test requiring randomness, it might make sense to instead define a fixture
```
def rng():
    return numpy.random.default_rng(42)
```

This has the downside of adding to the signature of each test, but it has the upside of being self-contained and not having to worry about global numpy random state.

Reply:
There is `pytest-rng`, but that's still using the legacy numpy random number API (and this is a trivial fixture to add anyway).

Reply:
> but it has the upside of being self-contained and not having to worry about global numpy random state.
Yes, this was my intention with the autouse fixture. We could use an explicit fixture instead.